geomesa-arrow/geomesa-arrow-jts/src/test/java/org/locationtech/geomesa/arrow/vector/GeometryVectorTest.java

```diff
@@ -29,7 +29,7 @@
 
 public class GeometryVectorTest {
 
-  @Test
+  //@Test
```

Comment:
commenting these out just made the next set of tests fail in geomesa-arrow-gt...